pyzeebe/client/client.py

```diff
@@ -49,7 +53,7 @@ async def run_process_with_result(
         variables: Optional[Dict] = None,
         version: int = -1,
         timeout: int = 0,
-        variables_to_fetch: Optional[List[str]] = None,
```

Comment:
Why delete this? This is a breaking change

Reply:
Sorry, this field doesn't exist in docs, I created bug: https://github.com/camunda/camunda-docs/issues/3367. Fixed.